nntrainer/layers/loss_layer.cpp

```diff
@@ -42,8 +42,33 @@ int LossLayer::initialize() {
   return status;
 }
 
+sharedConstTensors LossLayer::forwarding_with_val(sharedConstTensors in,
+                                         sharedConstTensors label) {
+
+  for(unsigned int i=0;i<num_inputs;++i){
+    net_input[i]->var = *in[i];
+  }
+
+  if (num_outputs == 0)
+    throw("invalid number of outputs");
```

Comment:
I guess throwing const char * is unintended
throw std::invalid_argument("");

Reply:
fixed.